frontend/src/routes/Governance/common/util.test.tsx

```diff
@@ -34,3 +34,94 @@ describe('Test resolveExternalStatus', () => {
     expect(resolveExternalStatus(policy)).toEqual(expected)
   })
 })
+
+describe('Test getPolicyRemediation', () => {
+  test('getPolicyRemediation should be enforce when all template-remediations = enforce', () => {
+    const rootPolicy: Policy = {
```

Comment:
Can you also one more test? root remediation: inform, policy template: enforce, informOnly